### PR TITLE
Add undocumented `CountryStatistics` model for `Rankings`

### PR DESCRIPTION
**Pull request by ItsMestro**
A list of `CountryStatistics` is returned when getting rankings with `RankingType.COUNTRY`

I don't know if implementation should be different to not make type hints awful but this at least allows for country rankings to be fetched.

### CODE REVIEW

File: ossapi/models.py [hunk 1]
```diff
@@ -1073,6 +1073,19 @@ class ChatMessage(Model):
...
+    created_at: Optional[Datetime]
+    display: int
+    id: int
+    mode: GameMode
+    performance: int
+    play_count: int
+    ranked_score: int
+    updated_at: Optional[Datetime]
+    user_count: int
+
```
Comment (tybug) [inline ossapi/models.py:1088]: By my count, this is what `CountryStatistics` looks like:

```python
class CountryStatistics(Model):
    code: str
    active_users: int
    play_count: int
    ranked_score: int
    performance: int
    country: Country
```

**Comment by ItsMestro** [comment]
Realized I didn't test this properly and does not work in its current form because the type instantiation doesn't like unions. But that's where my knowledge of this project ends.

**Review by tybug — COMMENTED** [review]
Thanks for the PR! You're right, we don't handle `Union` correctly right now. I'll implement this.

I think the typing here is OK. We don't have many good choices when the api returns a response like this. We *could* use something like [`override_types`](https://github.com/circleguard/ossapi/blob/bbde37f34466f95b7de64404033892ead28dd9da/ossapi/utils.py#L55), but depending on the user-passed `type` parameter instead. I think that's more complicated than is warranted right now, though.

File: ossapi/models.py (hunk 1 above)
Comment (ItsMestro) [reply ossapi/models.py:1088]: For having never looked at php before it was a good attempt. I trust you here way more than myself

**Comment by tybug** [comment]
I've touched things up - thanks for the contribution!

**Pull request merged**